src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Machine[] machines = new Machine[3];
+        machines[0] = new Bulldozer();
+        machines[1] = new Excavator();
+        machines[2] = new Truck();
+
+        for (Machine each : machines) {
```

Comment:
```suggestion
        for (Machine machine : machines) {
```

use meaningful naming, please